js/views/userPageVw.js

```diff
@@ -450,6 +456,7 @@ module.exports = Backbone.View.extend({
   renderItems: function (model) {
     "use strict";
     var self = this;
+    var select = document.getElementById('categories');
```

Comment:
It's better to do this.$el.find("#categories") to limit it to this view, just in case the same id is used somewhere else (ids shouldn't appear twice, but it's possible).